src/OrchardCore.Modules/OrchardCore.Lucene/Recipes/LuceneIndexStep.cs

```diff
@@ -44,6 +44,7 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                     {
                         luceneIndexSettings.Value.IndexName = luceneIndexSettings.Key;
                         await _luceneIndexingService.CreateIndexAsync(luceneIndexSettings.Value);
+                        await _luceneIndexingService.ProcessContentItemsAsync(luceneIndexSettings.Key);
```

Comment:
This can cause a request timeout which is why it was not added. In this case, if the recipe fails to execute then executing it a second time would probably not work.

Reply:
It should be a fire and forget type of execution here. Start the indexing background task for example.